src/discord-rest_request.c

```diff
@@ -360,7 +360,7 @@ discord_requestor_info_read(struct discord_requestor *rqtor)
 {
     int alive = 0;
 
-    if (CURLM_OK != curl_multi_socket_all(rqtor->mhandle, &alive))
+    if (CURLM_OK != curl_multi_socket_action(rqtor->mhandle, 0, 0, &alive))
```

Comment:
We need to check what `!= CURLM_OK` means for `curl_multi_socket_action()`. Does it actually translates to a `CCORD_CURLM_INTERNAL` error?

Reply:
![Screenshot_20240201_210120_Chrome](https://github.com/Cogmasters/concord/assets/73797168/6f602d8b-64e2-4b00-90d3-31b768841b81)

No documented change.